crates/jit/src/compiler.rs

```diff
@@ -158,7 +158,7 @@ impl Compiler {
             // show up be sure to log it in case anyone's listening and there's
             // an accidental bug.
             if relocations.len() > 0 {
-                log::info!("relocations found in trampoline for {:?}", sig);
+                log::debug!("relocations found in trampoline for {:?}", sig);
```

Comment:
This also seems higher priority than debug. This is "this should be super rare, and even though we have code to handle it, be aware that this super rare thing is happening!"

Reply:
This sounds like `warn` might be more appropriate then.

Reply:
This message in itself doesn't indicate that something went wrong, just that something we expect to be rare, but still valid, happened. I'll go with your judgement about whether this should be `info` or `warn` though.